### PR TITLE
Update FINOS lifecycle badge to ARCHIVED

### PR DESCRIPTION
**Pull request by mcleo-d**
This PR updates the Desktop Interoperability API Project FINOS lifecycle badge to INCUBATING inline with the project status reflected in FINOS.  

[FINOS - Incubating](https://finosfoundation.atlassian.net/wiki/display/FINOS/Incubating)

https://finosfoundation.atlassian.net/wiki/spaces/FINOS/pages/75530363/Incubating

## PR Scope Change
The scope of this PR has changed, as documented in https://github.com/finos-plexus/plexus-fdc3-samples/pull/19#issuecomment-738707136

### CODE REVIEW

**Comment by mcleo-d** [comment]
Hi @lspiro-Tick42 - Can a member of the API team review and merge this PR please? 👍 

Many thanks.

James.

**Comment by lspiro-Tick42** [comment]
I don't agree this is an incubating project.

We have an implementation of this API. DB have an implementation of the API on their existing Plexus Broker and are also (I believe) using as a primary API for their new broker.

I think we did agree to change the Repo name

**Comment by mcleo-d** [comment]
Thanks for the feedback @lspiro-Tick42.

Let's take the project through all the stories to shape and tidy that have been previously raised and listed below. We can then review the status of the project against [FINOS Activation Criteria](https://finosfoundation.atlassian.net/wiki/spaces/FINOS/pages/75530376/Activation). 

* [Remove "finos-plexus.github.io" from Plexus Interop metadata getHubRepos array](https://github.com/finos-plexus/plexus-interop/issues/122)
  * Please comment in the issue above that you're okay for this change to happen and cc @mcleo-d 

* [Confirm "finos-plexus.github.io" is appropriate repo name for Plexus Interop API Project](https://github.com/finos-plexus/plexus-interop/issues/120)
  * The repo name doesn't reflect the name of the project. 
  * Please comment in the issue above how the project should be represented. 
  * Can I suggest **Desktop Interoperability API** as reflected in FINOS metadata

* [Creation of 3 Plexus repositories for Plexus demo examples, migration of demo code and WhiteSource integration](https://github.com/finos-plexus/plexus-interop/issues/115)
  * We need to create 3 new repos for the demos held within the master project repo.
  * Please comment in the issue above what the names of the repos should be so I can schedule for creation in FINOS.

* [FINOS branding to be added to Plexus microsite : https://finos-plexus.github.io/plexus-interop/](https://github.com/finos-plexus/plexus-interop/issues/113)
  * This change needs to be done by a member of the Plexus team as noted in the issue above.

Many thanks,

James.

**Comment by mcleo-d** [comment]
@lspiro-Tick42 - I've assigned this to @maoo for review as the following epic (#24) has been created to take the Plexus Interop API Project into an ACTIVE state from INCUBATING 

I'm happy to discuss further on a future Plexus PMC call.

James.

**Comment by mcleo-d** [comment]
### Description
This pull request has been altered to change the Plexus Desktop API badge to `archived` as per issue #48. As a result, the name of the PR has been changed from **Update FINOS lifecycle badge to INCUBATING** to **Update FINOS lifecycle badge to ARCHIVED**

## Commits 
- update FINOS badge to archived aae5c34

## Markdown Change
```
# FINOS Plexus
[![FINOS - Incubating](https://cdn.jsdelivr.net/gh/finos/contrib-toolbox@master/images/badge-archived.svg)](https://finosfoundation.atlassian.net/wiki/display/FINOS/Archived)
```

## Label Change
[FINOS - Incubating](https://finosfoundation.atlassian.net/wiki/display/FINOS/Archived)

File: README.md [hunk 1]
```diff
@@ -1,4 +1,5 @@
 # FINOS Plexus
+[![FINOS - Incubating](https://cdn.jsdelivr.net/gh/finos/contrib-toolbox@master/images/badge-archived.svg)](https://finosfoundation.atlassian.net/wiki/display/FINOS/Archived)
```
Comment (maoo) [inline README.md:2]: It still says "FINOS - Incubating", should be "FINOS - Archived"

**Pull request merged**